Cakefile

```diff
@@ -440,3 +435,30 @@ task 'test:browser', 'run the test suite against the merged browser script', ->
   (-> eval source).call result
   testResults = runTests result.CoffeeScript
   process.exit 1 unless testResults
+
+task 'build:webpack', 'build webpack test bundle', ->
+  args = [
+    "./node_modules/webpack/bin/webpack.js"
+    "--entry=./test/webpack/entry.js", 
+    "--output-path=test/webpack",
+    "--output-filename=coffeescript.js"
+  ]
+
+  spawnNodeProcess args, 'both', (status) ->
+    process.exit(status)
+
+task 'test:webpack', 'run test suite against the webpack test bundle', ->
+  loader = """
+  var __exports__ = {};
```

Comment:
Webpack will scope known globals. It is not possible to access them directly. This, along with the custom entry file and the loader script in test:webpack task, allows the bundled CoffeeScript to escape thus it can be required in node environment.